exercises/space-age/.meta/generator/space_age_cases.rb

```diff
@@ -0,0 +1,31 @@
+class SpaceAgeCase < Generator::ExerciseCase
```

Comment:
There should be a `require 'generator/exercise_case'` at the top of this file.

See: https://github.com/exercism/xruby#implementing-a-generator